3rdparty/ngfx/ngfx.cmake

```diff
@@ -0,0 +1,67 @@
+option(NBL_BUILD_WITH_NGFX "Enable NGFX build" OFF)
+
+# NOTE: on windows default installation path is:
+# "C:/Program Files/NVIDIA Corporation/Nsight Graphics <version>/SDKs/NsightGraphicsSDK" <- define as "NGFX_SDK" environment variable
+# then you can pick SDK version with "NGFX_SDK_VERSION" cache variable (CMake GUI list supported)
+
+if(NBL_BUILD_WITH_NGFX)
+    if(NOT DEFINED ENV{NGFX_SDK})
+        message(FATAL_ERROR "\"NGFX_SDK\" environment variable must be defined to build with NBL_BUILD_WITH_NGFX enabled!")
+    endif()
```

Comment:
![image](https://github.com/user-attachments/assets/99f4e573-8a69-4aee-b236-d6cca473a72e)

What am I doing wrong @AnastaZIuk ?

Reply:
Wtf I need to define an envvar!? https://stackoverflow.com/questions/14343643/can-i-add-an-environment-variable-in-cmake-gui-wizard